python/requirements.txt

```diff
@@ -1,5 +1,5 @@
 Sphinx>=4.0.2
-pyarrow==14.0.0
+pyarrow==15.0.0
 pandas>=1.2.5
 opentelemetry-api>=1.0.0
-opentelemetry-sdk>=1.0.0
\ No newline at end of file
+opentelemetry-sdk>=1.0.0opentelemetry-sdk>=1.0.0
```

Comment:
Something got messed up here? Do we even use this file if CI still passes?

Reply:
We do and pip seems to do some magic knowing the name:
```
 Collecting opentelemetry-sdk>=1.0.0opentelemetry-sdk>=1.0.0
  Downloading opentelemetry_sdk-1.22.0-py3-none-any.whl (105 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 105.6/105.6 KB 11.3 MB/s eta 0:00:00
```

Reply:
I have no idea what happened and didn't see this when I sent in the PR. I'll push a fix.

Reply:
Fixed. Thanks for catching the mistake.